README.md

```diff
@@ -31,6 +31,14 @@ You will need to run them as admin. they create the user and database.
 `./build.sh test`
 
 ### Build Docker Image
+
+#### Setup Docker (OSX)
```

Comment:
`Setup boot2docker (OSX)` because we will want to add the instructions for `Docker Beta` which is the new tooling that replaces boot2docker - https://beta.docker.com/